src/mongocxx/test/gridfs/bucket.cpp

```diff
@@ -502,7 +502,7 @@ TEST_CASE("mongocxx::gridfs::downloader::read with arbitrary sizes", "[gridfs::d
 
     std::int64_t file_length = 100;
     std::int32_t chunk_size = 9;
-    std::int32_t read_size;
+    std::int32_t read_size = 0;
```

Comment:
Coverity seemed not to be able to see that we always assigned this in one of the sections. Fair enough. Init it to zero, and then require that it be non-zero before we start the test body.